programs/test/benchmark.c

```diff
@@ -643,6 +627,10 @@ int main( int argc, char *argv[] )
     memset( buf, 0xAA, sizeof( buf ) );
     memset( tmp, 0xBB, sizeof( tmp ) );
 
+    /* Avoid "unused static function" warning in configurations without
```

Comment:
Related: `ecc-heap.sh` doesn't run ECDSA benchmarks because they require SHA256. How about adding `MBEDTLS_SHA256_C` (and SHA224 until we make it optional) to the `ecc-heap.sh` config? (Which would incidentally mean that `ecc-heap.sh` wouldn't trigger the unused-function build error, but it's good to fix that anyway.)